visualization/app/codeCharta/ui/codeMap/rendering/geometryGenerator.ts

```diff
@@ -296,7 +294,7 @@ export class GeometryGenerator {
 		const floorSurfaceLabelMaterial = new MeshBasicMaterial({ map: labelTexture })
 		floorSurfaceLabelMaterial.needsUpdate = true
 		floorSurfaceLabelMaterial.side = DoubleSide
-		floorSurfaceLabelMaterial.transparent = true
+		floorSurfaceLabelMaterial.transparent = false // perfromance killer, why do we need it ?, if necessary try out with shader ?
```

Comment:
We use a Texture to write the folder name as a label on the floor. The idea was to add the transparency for the texture background, so that only the text is visible on the floor. Maybe we can improve this?